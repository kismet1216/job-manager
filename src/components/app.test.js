import React from 'react';
import ReactDOM from 'react-dom';
import App from './app';
import renderer from 'react-test-renderer';

describe('App', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('has a valid snapshot', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});


