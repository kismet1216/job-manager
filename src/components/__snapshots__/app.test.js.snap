// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App has a valid snapshot 1`] = `
Array [
  <header>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      面板
    </a>
    <a
      aria-current={null}
      href="/resume-maker"
      onClick={[Function]}
    >
      简历制作
    </a>
  </header>,
  <div
    className="board container-fluid"
  >
    <div
      className="procedures-container"
    >
      <div
        className="d-inline-block align-top"
      >
        <div>
          <button
            className="btn btn-outline-primary"
            onClick={[Function]}
          >
            <i
              className="fa fa-plus"
            />
             Add a new procedure
          </button>
        </div>
        <button
          className="btn btn-outline-primary "
          onClick={[Function]}
        >
          创建预定义模版
        </button>
      </div>
    </div>
  </div>,
]
`;
